### PR TITLE
Add Fortran language support

### PR DESCRIPTION
**Pull request by janmolnar**
See https://github.com/Glavin001/atom-beautify/issues/300

### CODE REVIEW

**Comment by Glavin001** [comment]
This is looking great! Just a few things to clean up. Also pull the latest upstream changes and then please run `npm run docs` as well. Everything looks good to merge after that! Thanks!

**Comment by Glavin001** [comment]
Looks good. Thank you!

**Comment by janmolnar** [comment]
I moved the files as suggested, pulled from upstream, and created the docs. Thanks for your helpful comments!

**Comment by Glavin001** [comment]
Published to v0.27.10

File: src/beautifiers/fortran-beautifier/index.coffee [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+"use strict"
+Beautifier = require('../beautifier')
+path = require("path")
+
+module.exports = class FortranBeautifier extends Beautifier
+    name: "Fortran Beautifier"
+
+    options: {
+        Fortran: true
```
Comment (Glavin001) [inline src/beautifiers/fortran-beautifier/index.coffee]: This should have been `false`.

**Pull request merged**